File: 1768-merge-strings-alternately/1768-merge-strings-alternately.java
class Solution {
    public String mergeAlternately(String word1, String word2) {
        int count = 0;
        String merged = "";
        while (count < word1.length() || count < word2.length()) {
            if (count < word1.length()) {
                merged = merged + word1.charAt(count);
            } 
            if (count < word2.length()) {
                merged = merged + word2.charAt(count);
            }
            count ++;
        }
        return merged;
    }
}

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        answer = []
        word1_list = list(word1)
        word2_list = list(word2)
        length1 = len(word1_list)
        length2 = len(word2_list)
        
        for i in range(min(length1, length2)):
            answer.append(word1_list[i])
            answer.append(word2_list[i])
        
        if length1 < length2:
            answer.extend(word2_list[length1:])
        else:
            answer.extend(word1_list[length2:])
        
        return ''.join(answer)